2021-01-14 20:01:51,209 INFO    MainThread:718751 [wandb_setup.py:_flush():70] setting env: {}
2021-01-14 20:01:51,209 INFO    MainThread:718751 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-14 20:01:51,209 INFO    MainThread:718751 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-14 20:01:51,209 INFO    MainThread:718751 [wandb_init.py:_log_setup():318] Logging user logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210114_200151-1fth1alt/logs/debug.log
2021-01-14 20:01:51,209 INFO    MainThread:718751 [wandb_init.py:_log_setup():319] Logging internal logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210114_200151-1fth1alt/logs/debug-internal.log
2021-01-14 20:01:53,026 INFO    MainThread:718751 [wandb_run.py:_console_start():1308] atexit reg
2021-01-14 20:01:53,027 INFO    MainThread:718751 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-14 20:01:53,027 INFO    MainThread:718751 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-14 20:01:53,027 INFO    MainThread:718751 [redirect.py:install():205] install start
2021-01-14 20:01:53,027 INFO    MainThread:718751 [redirect.py:install():220] install stop
2021-01-14 20:01:53,027 INFO    MainThread:718751 [redirect.py:install():205] install start
2021-01-14 20:01:53,027 INFO    MainThread:718751 [redirect.py:install():220] install stop
2021-01-14 20:01:53,027 INFO    MainThread:718751 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-14 20:02:06,955 INFO    MainThread:718751 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-14 20:02:06,956 INFO    MainThread:718751 [wandb_run.py:_restore():1250] restore
2021-01-14 20:02:06,956 INFO    MainThread:718751 [redirect.py:uninstall():224] uninstall start
2021-01-14 20:02:06,956 INFO    MainThread:718751 [redirect.py:_stop():279] _stop: stdout
2021-01-14 20:02:06,956 INFO    stdout    :718751 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-14 20:02:06,956 INFO    stdout    :718751 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-14 20:02:06,956 INFO    MainThread:718751 [redirect.py:_stop():285] _stop closed: stdout
2021-01-14 20:02:06,956 INFO    MainThread:718751 [redirect.py:_stop():291] _stop joined: stdout
2021-01-14 20:02:06,956 INFO    MainThread:718751 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-14 20:02:06,956 INFO    MainThread:718751 [redirect.py:uninstall():228] uninstall done
2021-01-14 20:02:06,956 INFO    MainThread:718751 [redirect.py:uninstall():224] uninstall start
2021-01-14 20:02:06,957 INFO    MainThread:718751 [redirect.py:_stop():279] _stop: stderr
2021-01-14 20:02:06,957 INFO    MainThread:718751 [redirect.py:_stop():285] _stop closed: stderr
2021-01-14 20:02:06,957 INFO    stderr    :718751 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-14 20:02:06,957 INFO    stderr    :718751 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-14 20:02:06,957 INFO    MainThread:718751 [redirect.py:_stop():291] _stop joined: stderr
2021-01-14 20:02:06,957 INFO    MainThread:718751 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-14 20:02:06,957 INFO    MainThread:718751 [redirect.py:uninstall():228] uninstall done
2021-01-14 20:02:07,434 INFO    MainThread:718751 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "e53aeacd24664e11b1155c472a349e18"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1155
      total_bytes: 1155
    }
  }
}

2021-01-14 20:02:09,441 INFO    MainThread:718751 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "0581e75974c3442d8c60430babe8c3c4"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 3848
      total_bytes: 3848
    }
  }
}

2021-01-14 20:02:10,485 INFO    MainThread:718751 [wandb_run.py:_show_files():1614] logging synced files
2021-01-14 20:02:10,486 INFO    MainThread:718751 [internal.py:handle_exit():139] Internal process exited
